src/screens/WebsocketResearchScreen/SignedChatScreen.tsx

```diff
@@ -61,7 +61,8 @@ const SignedChatScreen = () => {
       setLoading(true);
       await moveToAnonymousChannel({
         oldChannelId: selectedChannel?.id,
-        targetUserId: memberChat.user_id
+        targetUserId: memberChat.user_id,
+        source: 'userId'
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new `source` parameter is hardcoded as `'userId'`. This could limit the flexibility of your code if you want to use different sources in the future. Consider making this a variable or a constant that can be easily changed.
```diff
-        source: 'userId'
+        source: SOURCE_USER_ID
```
In this case, `SOURCE_USER_ID` should be defined in a constants file or at the top of your current file.


<!-- This is an auto-generated comment by OSS CodeRabbit -->